docs/docs/authentication/users.mdx

```diff
@@ -50,32 +50,32 @@ You're allowed to:
 
 ## Roles
 
-Each user can have one or multiple roles for API requests. You can see the roles of a user and set a default role in Nhost Console under **Users**.
+Each user has one **default role** and an array of **allowed roles**. These roles are used to resolve permissions for requests to [GraphQL](/graphql/permissions) and [Storage](http://localhost:3000/storage#permissions).
 
-Every GraphQL request is made with a specific role. This role will be used to resolve permissions when querying the database. In other words, every user can have multiple roles, but only one role will be applied for any given GraphQL request.
+When the user makes a request, only one role is used to resolve permissions. The default role is used if no role is explicitly specified. Users can only make requests using the default role or one of the allowed roles.
+
+You can manage users' default roles and allowed roles in the Dashboard at **Auth**.
 
 ### Default Role
 
-The default role is used when no role is specified in the GraphQL request. By default, users' default role is `user`.
+The default role is used when no role is specified in the request. By default, users' default role is `user`.
```

Comment:
Same sentence as the paragraph above "The default role is used if no role is explicitly specified." it reads repetitively, can we rephrase this or remove it?

Reply:
I think it's OK. One is a summary, another is specific. It's not always that users read docs from start to finish so sometimes it's worth duplicating this info IMO.